application/event_processor/event_processor.py

```diff
@@ -1,75 +1,198 @@
-from json import dumps
+from json import dumps, encoder
 from logging import getLogger
 from os import environ, getenv
+import json
```

Comment:
Why does json need to be imported twice?

Reply:
I may have been using it for other purposes and left it in here.